### PR TITLE
Add dependencies

### PR DESCRIPTION
**Pull request by leinadcoder**
Description for dependencies necessary for compile and configure vim

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,13 @@ Use this script to download, compile and configure Vim with just one command:
 ```
 ./do.sh
 ```
+
+### Dependencies
```
Comment (soonick) [inline README.md]: Remove on # and move to the top (right after # get-vim)

File: README.md [hunk 2]
```diff
@@ -5,3 +5,13 @@ Use this script to download, compile and configure Vim with just one command:
...
+
+ - ncurses-devel
+
+install depencies in fedora 24
+
+```sh
+sudo dnf install ncurses-devel # if you use sudo
```
Comment (soonick) [inline README.md]: Remove # if you use sudo

File: README.md [hunk 3]
```diff
@@ -5,3 +5,13 @@ Use this script to download, compile and configure Vim with just one command:
 ```
 ./do.sh
 ```
+
+### Dependencies
+
+ - ncurses-devel
+
+install depencies in fedora 24
```
Comment (soonick) [inline README.md]:
```
To install dependencies in Fedora:
```

File: README.md [hunk 4]
```diff
@@ -5,3 +5,13 @@ Use this script to download, compile and configure Vim with just one command:
 ```
 ./do.sh
 ```
+
+### Dependencies
+
+ - ncurses-devel
```
Comment (soonick) [inline README.md]: Remove, the command should be enough

**Comment by leinadcoder** [comment]
# get-vim

Dependencies

To install dependencies in Fedora:

``` sh
sudo dnf install ncurses-devel
```

Use this script to download, compile and configure Vim with just one command:

```
./do.sh
```

That's ok?

**Comment by soonick** [comment]
@leinadcoder looks good ;)

**Pull request merged**